repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/api-catalogue.yaml

```diff
@@ -694,17 +694,15 @@
             jcr:primaryType: hippostd:html
           common:FacetType: apispecification
           common:FullTaxonomy:
-          - health-services
-          - people-and-places
-          - data-and-information
-          - inpatients
-          - national-health-service
-          - hospital
-          - secondary-care
-          - demographics
+          - security
+          - patient-communication
+          - reference-data
+          - tests-and-diagnostics
```

Comment:
As this file is in the content directory, you will also need to make all of the changes in this file in the console too.

Reply:
Thanks for the feedback. As I checked, this two yaml file changes in the content directory are used only for the integration test run, not used at any functional level. They run off with the the local data.